app/models/like.rb

```diff
@@ -1,2 +1,9 @@
 class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def likes_counter(post)
+    array = post.likes
+    post.likes_counter = array.length
+  end
```

Comment:
**OPTIONAL**
Please use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#likes_counter` method every time a new post has been saved to the database, then make the `likes_counter` method private.